http_handler.go

```diff
@@ -3752,16 +3752,16 @@ func (h *Handler) handleReserveIDs(w http.ResponseWriter, r *http.Request) {
 
 	ids, err := h.api.ReserveIDs(req.Key, req.Session, req.Offset, req.Count)
 	if err != nil {
-		var esync ErrIDOffsetDesync
+		var esync IDOffsetDesyncError
```

Comment:
doesn't the pattern ErrIDOffsetDesync match the others in this PR?

Reply:
so an error variable should be `Err---`
an error type shoud be `---Error`